### DISCUSSION TITLE
[Question]What's the difference between KubeVela and Helm?

### DISCUSSION THREAD

**Discussion by Ophira0110** [kubevela/kubevela#911] [Q&A]

**Comment by ryanzhang-oss** [comment]
KubeVela relies on tools like Helm to package k8s operator/controllers.  However, Kubevela itself provides a lot more functionalities than merely packaging.  Please refer to [helm](https://kubevela.io/#/en/introduction?id=helm-etc) for more detailed descriptions.  

Here is a nutshell version

- Kubevela provides a standard model for applications which could comprise of multiple resources, each can come as a helm packaging
- Kubevela relies on the [Open Application Model](https://github.com/oam-dev) to support application operational needs. This is beyond the day one application install that helm tool excels at.
- Kubevela also provides a DevOps facing application model similar to the docker-compose files. This greatly simplified the way for users to run applications on Kubernetes.
- Kubevela also provides a powerful extensible way for platform builders to natively incorporate any new capabilities in the Kubernetes ecosystem.
